bin/install.sh

```diff
@@ -269,6 +266,9 @@ check_defaultpw () {
 
 check_defaultpw;
 
+# Pull down and install containers
+/home/pi/screenly/bin/upgrade_containers.sh
```

Comment:
Not sure about how `docker system prune` work. Since `upgrade_containers.sh` executes `docker compose up -d`, the containers should be up and running. Hypothetically, running containers shouldn't be removed after the `prune` command was executed. Disclaimer: Haven't tested this one by myself.

Reply:
Here are the man pages for`docker system prune`:
https://docs.docker.com/engine/reference/commandline/system_prune/

tl;dr is that it only deletes unused containers and volumes.

Reply:
Yeah I was just testing cause I thought maybe the containers were still starting when the command then ran, but after telling Nico I did three fresh installs with that prune commented out one time it worked then another time leaving the building of containers to the end also worked but then a final test didn't work but all I did different was say no to manage network but Nico said he tried it with saying yes to everything so I'll work on this more on weekend.. I was telling Nico we need to leave the PR just cause the issue is still active and we just have to change up the commits when we figure out the issue.